src/cloud/project/project-patch.md

```diff
@@ -90,14 +215,17 @@ To apply and test a custom patch:
    ```
 
    {:.bs-callout-info}
-   Make sure to test all patches in a pre-production environment.  For Magento Cloud, new branches can be created with     `magento-cloud environment:branch <branch-name>`
+   Make sure to test all patches in a pre-production environment. For Magento Cloud, new branches can be created with the `magento-cloud environment:branch <branch-name>` CLI command.
+
+## Apply patches to a non-Cloud project {#standalone}
```

Comment:
@meker12, I agree with your concern. It may look a bit out of place and contain redundant information, so it would be nice to have just a link to non-Cloud MQP documentation.

Reply:
Thanks @meker12 & @thiaramus. Removed procedure and kept a link to the MQP topic in 08ecccf3f30e8d3ea53186c61a239c067d0b0c9b